codec.proto

```diff
@@ -85,12 +85,7 @@ message ConsensusParams {
   VersionParams   version   = 4 [(gogoproto.nullable) = false];
 }
 
-message Data { repeated Tx txs = 1; }
-
-message Tx {
-  bytes tx_hash = 1;
-  bytes tx_raw  = 2;
-}
+message Data { repeated bytes txs = 1; }
```

Comment:
Do we even need this any more?

Reply:
Are you suggesting that we get rid of the transaction payload from `EventBlock` and keep it only in `EventTx`?